Microwave.App/Program.cs

```diff
@@ -30,7 +31,7 @@ static void Main(string[] args)
 
             CookController cooker = new CookController(timer, display, powerTube);
 
-            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, door, display, light, buzzer, cooker);
```

Comment:
Notice that buzzer has been removed, and might need to be added again

Reply:
Has been changed now.